providers/base/websocket/handlers/ws_query_handler.go

```diff
@@ -159,6 +159,9 @@ func (h *WebSocketQueryHandlerImpl[K, V]) start() error {
 		return errors.ErrDialWithErr(err)
 	}
 
+	// Wait for the connection timeout before sending the initial payload(s).
+	time.Sleep(h.config.PostConnectionTimeout)
```

Comment:
we can get rate-limited after the first connection?

Reply:
no messages should have been sent to the connection so should not be possible on init. it is possible if the connection is multi-plexed but as long as the multi-plexing is spaced out sufficiently we shouldnt see this issue. Ideally the time between spinning up concurrent connections is greater than the rate limit configs.